stub.py

```diff
@@ -92,7 +92,8 @@ def get_seattraits(self, event_id, params={}):
         seats = requests.get(url, headers=self.headers)
         if seats.status_code != 200:
             error_msg += f'\n{seats.status_code} {seats.reason}'
-            return
+            print(error_msg)
+            return False
```

Comment:
Please could you return `{}` here @hobzee? The reason for that is because we always want to return a dictionary (so that we don't get type conflicts) from these functions and an empty dictionary will evaluate to `False` anyway.

Cheers bro.